### DISCUSSION TITLE
Migration Guide from v2 to v3?

### DISCUSSION THREAD

**Discussion by SammyIsra** [hubotio/hubot#1595] [Q&A]
Hi all,

I upgraded Hubot from v2 to v3 as an attempt to fix NPM Audit issues, however, since then I have not been able to run `npm ./bin/hubot` to run the project, as I get the error: 

```
C:\<path to project root>\bin\hubot:5
npm install
    ^^^^^^^

SyntaxError: Unexpected identifier
    at new Script (vm.js:51:7)
    at createScript (vm.js:136:10)
    at Object.runInThisContext (vm.js:197:10)
    at Module._compile (module.js:626:28)
    at Object.Module._extensions..js (module.js:673:10)
    at Module.load (module.js:575:32)
    at tryModuleLoad (module.js:515:12)
    at Function.Module._load (module.js:507:3)
    at Function.Module.runMain (module.js:703:10)
    at startup (bootstrap_node.js:193:16)
```

Additionally, running `npm install` show the following peer dependency warnings:

```
npm WARN hubot-diagnostics@0.0.1 requires a peer of hubot@2.x but none is installed. You must install peer dependencies yourself.
npm WARN hubot-help@0.1.3 requires a peer of hubot@2.x but none is installed. You must install peer dependencies yourself.
npm WARN hubot-maps@0.0.2 requires a peer of hubot@2.x but none is installed. You must install peer dependencies yourself.
npm WARN hubot-redis-brain@0.0.3 requires a peer of hubot@2.x but none is installed. You must install peer dependencies yourself.
```

Before trying to fix those errors, I wanted to try my hand in finding some guidance here. 

I haven't found any resources on how to upgrade to v3, or what's really different about it, so my attempt to upgrade was really brute force-ish.

For an example of what versions changed, I have the following request open on my own project: https://github.com/SammyIsra/fred/pull/27

**Comment by joeyguerra** [comment]
I think you're supposed to just execute `bin/hubot` to start the app.
